README.md

```diff
@@ -2,31 +2,48 @@
 
 ## General Remarks
 
-**Please note that this software is still in a very early state of development. Its existing functionality hasn't been properly tested yet.**
+**Please note that this software is still in a early state of development. 
+Currently, only the Gregorian and Julian calendar are supported.**
 
 ## Introduction
 
-`JDNConvertibleCalendar` offers a convenient way to convert a given date to different calendar formats making use of Julian Day Number (JDN). 
+`JDNConvertibleCalendar` offers a convenient way to convert a given date to other calendars making use of the *Julian Day*. The Julian Day is
+the continuous count of days since the beginning of the Julian Period in 4713 BCE.
+The Julian Day Count (JDC) contains a fraction that represents the daytime. A fraction of .5 represents midnight and .0 represents noon. 
+For example, the day with the date January 27th 1987 starts with the JDC of 2446822.5 and ends with a JDC of 2446823.4999… Hence, noon is at 2446823.0.
+The corresponding Julian Day Number (JDN) is at noon, which is 2446823 for the given example (January 27th 1987).
 
-Currently, Gregorian and Julian calendar are supported.
+For more information about the Julian Day, see Jean: Meeus, Astronomical Algorithms, 1998, 59pp. and <https://en.wikipedia.org/wiki/Julian_day>
```

Comment:
For more information about the Julian Day, see: Jean Meeus, Astronomical Algorithms, 1998, 59pp. and <https://en.wikipedia.org/wiki/Julian_day>

Reply:
fixed in 
https://github.com/dhlab-basel/JDNConvertibleCalendar/pull/4/commits/393b286486dd8f83c3f6245f5e4e88e3b4e2ce98